be/src/exec/pipeline/set/except_context.cpp

```diff
@@ -11,7 +11,7 @@ Status ExceptContext::prepare(RuntimeState* state, const std::vector<ExprContext
 
     _dst_tuple_desc = state->desc_tbl().get_tuple_descriptor(_dst_tuple_id);
     _dst_nullables.reserve(build_exprs.size());
-    for (auto build_expr : build_exprs) {
+    for (const auto& build_expr : build_exprs) {
```

Comment:
The type of `build_exprs` is `const std::vector<ExprContext*>`.  

So I think the type of `build_expr` is `ExprContext*`. The cost of copy pointers is affordable.

Reply:
If it's a pointer the cost is really small